types/aws-sdk2-types/lib/credentials/saml_credentials.d.ts

```diff
@@ -0,0 +1,34 @@
+import {Credentials} from '../credentials';
+export class SAMLCredentials extends Credentials {
+		/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/aws-sdk2-types/lib/credentials/saml_credentials.d.ts;
  Line: 3;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.